### ISSUE TITLE
Make privacy policy link sufficiently distinguishable

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#4369]
### What should we do?

Link to privacy policy is distinguished only via color which is insufficient. We could reuse the style from the What's new popup.

<img width="1136" alt="image" src="https://github.com/camunda/camunda-modeler/assets/28307541/0e19b0fb-dde1-4e7d-b8cd-f1ad54cfa0bd">

### Why should we do it?

Child of https://github.com/bpmn-io/internal-docs/issues/909
